docs/Basics/download-project.md

```diff
@@ -5,13 +5,11 @@
 If you ever need to save the contents of your project without pushing to a Git repository,
 you can download a zip file of the current project directly from the Studio settings menu.
 
-![Download a Project](../../assets/images/download-project-as-zip.gif)
```

Comment:
Alt tags are generally good unless you're repeating things said elsewhere, or otherwise adding something which is not beneficial to the screenreader. Repeating the header is redundant, and describing the actions of the screencast would also be redundant because we describe them below, so in this case it's best to simply let the image be invisible to screenreaders.